Resources/Prototypes/Entities/Structures/Doors/Airlocks/assembly.yml

```diff
@@ -195,6 +195,25 @@
     sprite: DeltaV/Structures/Doors/Airlocks/Glass/medical.rsi #Delta V - Resprite Doors
     state: "assembly"
 
+#Chemistry
+- type: entity
+  parent: AirlockAssembly
+  id: AirlockAssemblyChemistry
+  suffix: Chemistry
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Doors/Airlocks/Standard/chemistry.rsi #Delta V - Resprite Doors
+    state: "assembly"
+
+- type: entity
+  parent: AirlockAssembly
+  id: AirlockAssemblyChemistryGlass
+  suffix: Chemistry, Glass
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Doors/Airlocks/Glass/chemistry.rsi #Delta V - Resprite Doors
+    state: "assembly"
+
```

Comment:
move these to a file in deltav folder since this doesnt exist upstream

Reply:
Wait what, chemistry and hydroponics don't exist upstream?